### ISSUE TITLE
Implement support for Conflict resolution

### ISSUE THREAD

**Issue by NateRickard** [Azure/azure-sdk-for-android#18]
Once design is complete (issue #17).  This issue is a placeholder for implementing that design (will likely be broken out into several additional issues).

**Comment by bsiegel** [comment]
Closing as wontfix, we are preparing for a release of the Track 2 SDK libraries which are based on a different codebase.
